test/e2e/info_test.go

```diff
@@ -202,11 +205,22 @@ var _ = Describe("Podman Info", func() {
 		}
```

Comment:
May I ask for one more case here? Append:
```
    // just because we requested sqlite doesn't mean it stays that way. 
    // once a boltdb exists, podman will forevermore stick with it
    arg: "", want: "boltdb"},
```

Reply:
Sure